components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesHistoryStorage.kt

```diff
@@ -102,7 +103,7 @@ open class PlacesHistoryStorage(
 
     override suspend fun getTopFrecentSites(numItems: Int): List<TopFrecentSiteInfo> {
         return withContext(readScope.coroutineContext) {
-            places.reader().getTopFrecentSiteInfos(numItems).map { it.into() }
+            places.reader().getTopFrecentSiteInfos(numItems, frecencyThreshold = FrecencyThresholdOption.NONE).map { it.into() }
```

Comment:
This fixed compatibility with an unrelated change that's in appservices main branch. We should probably pull it out into a separate PR.

Reply:
Up to you on putting it into a separate PR. I don't have any strong opinion on it sneaking in here, really.